code/game/machinery/doors/door.dm

```diff
@@ -222,6 +222,10 @@
 /obj/machinery/door/proc/try_to_crowbar(obj/item/I, mob/user)
 	return
 
+/obj/machinery/door/proc/try_to_crowbar_secondary(obj/item/I, mob/user)
+	try_to_crowbar(I, user)
```

Comment:
Honestly, just make it not do anything if it's not defined. The only reason right-click passes left-click is because of the fact that BYOND sends it all down the same chain, and we don't check it very often.